internal/apps/dop/endpoints/release.go

```diff
@@ -30,6 +30,7 @@ import (
 	"github.com/erda-project/erda-proto-go/dop/projectpipeline/pb"
 	"github.com/erda-project/erda/apistructs"
 	"github.com/erda-project/erda/internal/apps/dop/providers/projectpipeline"
+	pipelineaction "github.com/erda-project/erda/internal/apps/dop/providers/rule/actions/pipeline"
```

Comment:
`pipelineaction` has another meaning, represent for `pipeline action(job)`.
use another suitable name.

Reply:
done